akka-persistence-query/src/main/scala/akka/persistence/query/journal/leveldb/EventsByPersistenceIdPublisher.scala

```diff
@@ -181,7 +181,7 @@ private[akka] class CurrentEventsByPersistenceIdPublisher(
     deliverBuf()
     if (highestSeqNr < toSequenceNr)
       toSeqNr = highestSeqNr
-    if (highestSeqNr == 0L || (buf.isEmpty && currSeqNo > toSequenceNr))
+    if (buf.isEmpty && (currSeqNo > toSequenceNr) || currSeqNo == fromSequenceNr)
```

Comment:
I'd find it more readable as:

    if (buf.isEmpty && (currSeqNo > toSequenceNr || currSeqNo == fromSequenceNr))